sklearn/tree/export.py

```diff
@@ -733,6 +735,11 @@ def export_graphviz(decision_tree, out_file=None, max_depth=None,
         Number of digits of precision for floating point in the values of
         impurity, threshold and value attributes of each node.
 
+    fontname : str,optional (default='helvetica')
+        Set dot file "font name" field
+        Output font cannot display optional
```

Comment:
I don't understand this description. Fine to just remove this sentence

Reply:
I'm so sorry ^_^!  My English is not so good

Reply:
No worries! You're making a much better attempt than I am at most contributors' native languages!

Reply:
I have modified it.